zaberMotionSupport/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../
+include $(TOP)/configure/CONFIG
+#----------------------------------------
+#  ADD MACRO DEFINITIONS AFTER THIS LINE
+#=============================
+T_A = $(EPICS_HOST_ARCH)
+
+ifndef ZML_HOST_ARCH
+$(error "motorZaberMotion: Unsupported architecture: $(T_A)")
+endif
+
+LIB_INSTALLS += $(wildcard ZaberMotionCppSupport/lib/$(ZML_HOST_ARCH)/*)
+
+#=============================
+
+include $(TOP)/configure/RULES
```

Comment:
I found out what is causing the `make clean` command to remove all files in `zaberMotionSupport`. The issue is that for whatever reason the make system is executing the target `build_clean` defined in `epics-base/configure/RULES_BUILD`:

```makefile
build_clean:
	$(ECHO) "Cleaning"
	$(RM) *.i *$(OBJ) *.a \
	$(LIBNAME) $(TESTLIBNAME) $(SHRLIBNAME) $(TESTSHRLIBNAME) \
	$(DLLSTUB_LIBNAME) $(TESTDLLSTUB_LIBNAME) \
	$(LOADABLE_SHRLIBNAME) \
	$(INC) $(TARGETS) $(TDS) $(CLEANS) \
	*.out MakefileInclude *.manifest *.exp \
	$(COMMON_INC) $(HDEPENDS_FILES) $(PRODTARGETS) $(TESTPRODTARGETS) \
	$(TESTSCRIPTS) $(TAPFILES) $(JUNITFILES)
ifdef RES
	@$(RM) *$(RES)
endif
```

The problem is that OBJ isn't defined and so it winds up running:
```bash
rm -f *.i * *.a *.out MakefileInclude *.manifest *.exp
```

`rm -f *` attempts to remove all files in a directory.

Reply:
If you define OBJ in this Makefile, it won't be deleted on clean but that it is a total hack

Reply:
Ok the issue was that in `zaberMotionSupport` I was included `$(TOP)/configure/RULES`, which includes `epics-base/configure/RULES`, AND I had defined `T_A`, so in the end I was pulling in `epics-base/configure/RULES_BUILD`, which defines `clean` this way. I copied this from the ADPointGrey support folder without understanding what it meant.

Changing this to include `$(TOP)/configure/RULES_DIRS`, or simply not including any rules and defining a custom clean target works fine.